tests/test_tet.py

```diff
@@ -290,68 +290,68 @@ def test_ordinal(self):
         )
 
     def test_currency_integer(self):
-        self.assertEqual(self.n2w.to_currency(1.00), 'ida euru')
-        self.assertEqual(self.n2w.to_currency(2.00), 'rua eurus')
-        self.assertEqual(self.n2w.to_currency(3.00), 'tolu eurus')
-        self.assertEqual(self.n2w.to_currency(4.00), 'haat eurus')
-        self.assertEqual(self.n2w.to_currency(5.00), 'lima eurus')
-        self.assertEqual(self.n2w.to_currency(6.00), 'neen eurus')
-        self.assertEqual(self.n2w.to_currency(7.00), 'hitu eurus')
-        self.assertEqual(self.n2w.to_currency(8.00), 'walu eurus')
-        self.assertEqual(self.n2w.to_currency(9.00), 'sia eurus')
-        self.assertEqual(self.n2w.to_currency(10.00), 'sanulu eurus')
-        self.assertEqual(self.n2w.to_currency(11.00), 'sanulu resin ida eurus')
-        self.assertEqual(self.n2w.to_currency(12.00), 'sanulu resin rua eurus')
-        self.assertEqual(self.n2w.to_currency(13.00), 'sanulu resin tolu eurus')
-        self.assertEqual(self.n2w.to_currency(14.00), 'sanulu resin haat eurus')
-        self.assertEqual(self.n2w.to_currency(15.00), 'sanulu resin lima eurus')
-        self.assertEqual(self.n2w.to_currency(16.00), 'sanulu resin neen eurus')
-        self.assertEqual(self.n2w.to_currency(17.00), 'sanulu resin hitu eurus')
-        self.assertEqual(self.n2w.to_currency(18.00), 'sanulu resin walu eurus')
-        self.assertEqual(self.n2w.to_currency(19.00), 'sanulu resin sia eurus')
-        self.assertEqual(self.n2w.to_currency(20.00), 'ruanulu eurus')
+        self.assertEqual(self.n2w.to_currency(1.00), 'ida dólar')
+        self.assertEqual(self.n2w.to_currency(2.00), 'rua dólares')
```

Comment:
Is dólar and dólares really good Tetum?
I don't know the answer, but I have not seen it before.

Reply:
In Tetun we say **_Dolar ida_**